packages/portfolio-contract/tools/plan-solve.ts

```diff
@@ -426,10 +445,24 @@ export const rebalanceMinCostFlowSteps = async (
           break;
         }
         // XXX: revisit https://github.com/Agoric/agoric-sdk/pull/11953#discussion_r2383034184
-        case 'poolToEvm':
+        case 'poolToEvm': {
+          const protocol = getProtocolFromPlace(edge.src);
```

Comment:
We should never need to heuristically infer yield protocol. If `PoolPlaces` is the source of truth, then let's just read from there.
```suggestion
          // @ts-expect-error edge.src must be in PoolPlaces
          const protocol = PoolPlaces[edge.src].protocol;
```